snippets/csharp/tour/attributes/Program.cs

```diff
@@ -31,6 +31,29 @@ public class Program
     {
         public static void Main(string[] args)
         {
+            Type widgetType = typeof(Widget);
```

Comment:
Let's add a marker here (and at the end) rather than using line numbers. Line numbers are very brittle in the OPS build.  For example:

```csharp
// <SnippetReadAttributes>
```